frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "coverage": "jest --coverage .",
     "eslint": "eslint --ext .js,.jsx,.ts,.tsx,.json,.gql,.graphql --color",
     "lint": "NODE_OPTIONS=--max-old-space-size=4096 yarn eslint .",
+    "gherkin-lint": "./node_modules/.bin/gherkin-lint -c ./.gherkin-lintrc ./packages/dev-console/integration-tests/features",
```

Comment:
Your readme does not specify to run this command. Perhaps put it under [Generic Standards](https://github.com/openshift/console/blob/e0158df322c995c0cd153423a69c5743ba3f708d/frontend/packages/dev-console/integration-tests/README.md#generic-standards)?

Reply:
updated